fastcomplete/__init__.py

```diff
@@ -15,9 +19,10 @@ def cached_complation_finder(completion_finder_cls):
 
     @wraps(completion_finder_cls, updated=())
     class CachedCompletionFinder(completion_finder_cls):
-        def __call__(self, *args, **kwargs):
-            parser_cache.save_cache(args, kwargs)
-            return super().__call__(*args, **kwargs)
+        def __call__(self, parser, *args, **kwargs):
```

Comment:
Maybe rename parser to the name of the argument within argcomplete.CompletionFinder

Reply:
ok